translation/message_format.rst

```diff
@@ -263,6 +263,69 @@ Usage of this string is the same as with variables and select::
             }
         }
 
+Ranges
+~~~~~~
+
+Sometimes, it can be handy to use ranges for a translation. The intl component provide a way to do this, using the ``choice`` function.
```

Comment:
```suggestion
Sometimes, it can be handy to use ranges for a translation. The ``intl`` component provides a way to do this, using the ``choice`` function.
```

Reply:
fixed